config/settings/local.py

```diff
@@ -119,7 +119,8 @@
     "localhost:8080",
     "0.0.0.0",
 ]
-
+METACI_LONG_RUNNING_BUILD_CLASS = "metaci.build.autoscaling.LocalOneOffBuilder"
```

Comment:
I would probably put this in base.py so that we can test the production settings locally. We've occasionally had reason to do that.

Reply:
How does putting this in base.py help us test the production settings locally?

Reply:
If you run `DJANGO_SETTINGS_MODULE="config.settings.production" ./manage.py runserver` locally, this line will not run because there's no HEROKU_TOKEN or HEROKU_APP_NAME: https://github.com/SFDO-Tooling/MetaCI/pull/2042/files#diff-b547313c08ca0949c654ff88d1c7f07a56d0f6c3076fa8152ab1243ca402fccaR260

So METACI_LONG_RUNNING_BUILD_CLASS will be unset and trying to run a one-off build will fail. If we move this line to base.py, then it would inherit the default and run it in a subprocess.

Reply:
I don't feel strongly about this one. We can change it later if we need it.